jordinner/src/pages/restaurants/signup.jsx

```diff
@@ -1,9 +1,8 @@
-import React, {useState, useEffect} from 'react'
-import Link from 'next/link'
+import React, { useState, useEffect } from 'react';
```

Comment:
All changes in sign up are visual and nothing coherent requested other than the comment, keep the code as is as it's not related to sign in